ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.5.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.5"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: openshift-ci/build-root/Dockerfile
+  from: bin
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: oc-bin-image
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+  bin:
+    requests:
+      cpu: "2"
+      memory: 8Gi
+  e2e-aws:
+    requests:
+      cpu: "4"
+      memory: 4Gi
+tag_specification:
+  name: "4.5"
+  namespace: ocp
+test_binary_build_commands: make test
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-test
+      commands: ./bin/extended-platform-tests run openshift/conformance/parallel --run
+        "High-24829"
```

Comment:
Does this mean that we are only testing a single test for 4.4 and 4.5?

Reply:
@shawn-hurley Yes, this repo is private for the QE. But, the logs are public. We don't want to output any sensitive logs, so just add test one case here. And, here, we just want to make sure the generated binary `extended-platform-tests` works.